packages/block-library/package.json

```diff
@@ -27,7 +27,7 @@
 	"sideEffects": [
 		"build-style/**",
 		"src/**/*.scss",
-		"{src,build,build-module}/*/init.js"
+		"{src,build,build-module}/**/init.js"
```

Comment:
This seems to be a bug in Gutenberg - to match nested directories you want to use `**`.

Reply:
According to the library that webpack uses: https://github.com/micromatch/micromatch

Reply:
I don’t think it’s a bug, as there are currently no nested directories to match. More info on why there is `init.js` file for every core block in https://github.com/WordPress/gutenberg/tree/trunk/packages/block-library#registering-individual-blocks.

Reply:
I see! This means that no file can be imported just for its side-effects, right? 

```js
// This doesn't work
import './some-file-with-sideeffects';
```

And the right approach would be to wrap the side effects in a `init` function and import and execute that function, right?

```js
import { init } from './some-file-with-sideeffects';

init();
```

Reply:
If you have a file with side effects like `init()` call for the runtime, you need to list the file where the usage is as explained in https://github.com/WordPress/gutenberg/tree/trunk/packages#optimizing-for-bundlers. Importing itself doesn't resolve the issue because webpack will aggressively remove all the code not exported when using `sideffects` in `package.json`.